Documents/Azure Network automation/1.How-To-Start-NETWORK-Cycle.txt

```diff
@@ -41,11 +41,11 @@ Edit XML file :
 				<OsImage>Distro_OS_Image_Name_As_Appearing_under_Azure_OS_Images</OsImage>
 			</Distro>
 		3.Save file.
-		
+
 Start Automation:
 	1.	On powershell in Administrator mode, go to automation parent folder.
 	2.	Execute following command
-	
+
 	.\AzureAutomationManager.ps1 -xmlConfigFile .\Azure_ICA_all.xml -cycleName NETWORK -Distro YourDistroName -runtests
```

Comment:
should we also delete this file? the command is outdated

Reply:
that whole folder requires a lot of cleanup and update, didn't want to get into too much changes there for now.